tests/test_deprecated.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2014-2016 Canonical Limited.
```

Comment:
Would it be more useful to have singular LICENSE and COPYRIGHT files in the root of the repository, and replace the copyright statement in the header of every file with a reference to that? Otherwise, we end up having to touch every file, every year, to keep it up to date, rather than just the two in the root. We'd need to research the license requirements and see if there are any recommended best practices for this.